pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -274,7 +274,7 @@ func TestPriorityQueue_AddUnschedulableIfNotPresent(t *testing.T) {
 // current scheduling cycle will be put back to activeQueue if we were trying
```

Comment:
Do you mind revisiting this comment? I think it's missing a period.

Reply:
Updated.